src/ILCompiler.WebAssembly/src/ILCompiler.WebAssembly.csproj

```diff
@@ -9,6 +9,7 @@
     <TargetFramework>netstandard1.3</TargetFramework>
     <NoWarn>8002</NoWarn>
     <OutputPath>$(BaseOutputPath)$(OSPlatformConfig)/tools</OutputPath>
+    <LangVersion>7.2</LangVersion>
```

Comment:
We set the LangVersion centrally in dir.props. I assume you're doing this because VS 2017 is broken. This is an unfortunate point in time issue due to us upgrading to C# 8.0 and the insistence of the C# language team that opting into C# 8.0 needs to be a complicated a two step process where you **both** choose the latest beta package of the compiler **and** set the LangVersion to the value of `preview` that no old compiler (or VS 2017) understands. We don't actually need 8.0; only a single file in CoreLib needs it so far.

This will get cleared up once the C# team decides it's time to end inflicting unnecessary pain on users and `Latest` == 8.0. My understanding is that it will happen soon.

Reply:
Yes, VS2017 was complaining about some 7.1 features like `LLVMValueRef castReturnAddress = default;`.  I'll revert that.